core/app/[locale]/(default)/(faceted)/brand/[slug]/page-data.ts

```diff
@@ -16,6 +16,11 @@ const BrandPageQuery = graphql(`
         }
       }
       settings {
+        inventory {
+          defaultOutOfStockMessage
```

Comment:
Is this translatable in the CP? Mostly curious since I don't know much about this!